taskcluster/ci/test/kind.yml

```diff
@@ -55,7 +55,10 @@ jobs:
         attributes:
             build-type: debug
         treeherder:
-            symbol: ui
+            kind: test
+            platform: 'ui-test/opt'
+            symbol: debug(ui-test-x86)
```

Comment:
Oh, I just noticed the decision task is broken. You need to add `debug` in this list: https://github.com/mozilla-mobile/reference-browser/blob/ab2f58f70ab0f1d9a7cd515ae5bede985264fda6/taskcluster/ci/config.yml#L5-L10